301-subnet-driven-deployment/azuredeploy.json

```diff
@@ -0,0 +1,299 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "domainName": {
+      "type": "string",
+      "defaultValue": "fabrikam.com",
+      "metadata": {
+        "description": "Domain to create for the Lab"
+      }
+    },
+    "assetLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The location of resources such as templates and DSC modules that the script is dependent"
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/301-subnet-driven-deployment/"
+    },
+
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "zeAdmin",
+      "metadata": {
+        "description": "Admin username"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password"
+      }
+    },
+    "StorageAccountPrefix": {
+      "type": "string",
+      "defaultValue": "st",
+      "metadata": {
+        "description": "Storage Account Prefix (prepended to the unique id generated"
```

Comment:
Do it the other way. Prepend the unique id to a suffix. It helps spread the storage accounts on different clusters rather than clustering them on the same cluster due to same prefixes. 

Since the amount of characters in prefix could cause a storage account name failure, I suggest removing this and make this a variable like in #944